.travis.yml

```diff
@@ -30,9 +30,9 @@ install:
       export PATH=/Users/travis/miniconda3/bin:$PATH
 
       conda update --yes conda
+      conda config --set show_channel_urls True
```

Comment:
See this PR ( https://github.com/conda-forge/conda-smithy/pull/137 ). I went ahead and put it before the update step.